doc/classes/TreeItem.xml

```diff
@@ -433,6 +474,7 @@
 			<argument index="1" name="texture" type="Texture">
 			</argument>
 			<description>
+				Sets the icon [Texture] for column [code]column[/code].
```

Comment:
Avoid "of the" constructions where possible:

Sets the given column's icon [Texture].